package.json

```diff
@@ -31,6 +31,7 @@
     "cross-fetch": "3.1.6",
     "elliptic": "6.5.4",
     "ethers": "6.9.0",
+    "react-native-passkey": "^2.1.1",
```

Comment:
I remember 2 libraries were mentioned, `react-native-passkey` vs [`react-native-passkeys`](https://github.com/peterferguson/react-native-passkeys), we prefer this one? just checking before we commit to the lib I guess. seems like this one has more stars on github. that's the only metrics I can go by knowing nothing about either libs and passkey on mobile in general.